shell/platform/android/io/flutter/view/AccessibilityBridge.java

```diff
@@ -936,9 +960,13 @@ void updateRecursively(float[] ancestorTransform, Set<SemanticsObject> visitedOb
             assert globalTransform != null;
             assert globalRect != null;
 
+            if (routeName != null) {
+                routeNames.add(routeName);
```

Comment:
since you're only reading the last node in the list, you only need to remember the last string you've seen, not all of them. so you should be able to do away with the list entirely.

ditto on the iOS side

Reply:
Because of my original lack of `retain` and `release` the list version lasted _slightly_ longer before crashing on the iOS side - I'll clean that up.

Reply:
Done